google/cloud/pubsub/publisher_connection_test.cc

```diff
@@ -334,6 +340,60 @@ TEST(PublisherConnectionTest, HandleTransientEnabledRetry) {
   EXPECT_EQ("test-message-id-0", *response);
 }
 
+#ifdef GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
+using ::google::cloud::testing_util::DisableTracing;
+using ::google::cloud::testing_util::EnableTracing;
+using ::google::cloud::testing_util::InstallSpanCatcher;
+using ::google::cloud::testing_util::SpanNamed;
+using ::testing::Contains;
+using ::testing::IsEmpty;
+
+TEST(MakePublisherConnectionTest, TracingEnabled) {
+  auto span_catcher = InstallSpanCatcher();
+  auto mock = std::make_shared<pubsub_testing::MockPublisherStub>();
+  Topic const topic("test-project", "test-topic");
+  EXPECT_CALL(*mock, AsyncPublish)
+      .WillOnce([&](google::cloud::CompletionQueue&, auto,
+                    google::pubsub::v1::PublishRequest const& request) {
```

Comment:
compiler is mad at you for naming `request` here and below

Reply:
the compiler got smarter.... older code does it and doesn't get yelled at 😢 

fixed.